mlflow/store/artifact/databricks_artifact_repo.py

```diff
@@ -231,6 +238,38 @@ def _azure_upload_file(self, credentials, local_file, artifact_path):
         except Exception as err:
             raise MlflowException(err)
 
+    def _azure_adls_gen2_upload_file(self, credentials, local_file, artifact_path):
+        """
+        Uploads a file to a given Azure storage location using the ADLS gen2 API.
+        """
+        try:
+            headers = self._extract_headers_from_credentials(credentials.headers)
+
+            # try to create the file
+            try:
+                put_adls_file_creation(credentials.signed_uri, headers=headers)
+            except requests.HTTPError as e:
+                raise e
+
+            # next try to patch the file
+            offset = 0
+            for chunk in yield_file_in_chunks(local_file, _AZURE_MAX_BLOCK_CHUNK_SIZE):
```

Comment:
Should we be using a different chunk size for ADLS?

cc @mswit-databricks

Reply:
We use the same size chunks in the cloudstorage code in universe so I am assuming this is fine.

Reply:
What is the default now? Generally, using the same size is fine.